src/ContactPage/Contact.js

```diff
@@ -92,6 +100,7 @@ class Contact extends React.Component {
               placeholder="Name"
               maxLength="20"
               autoComplete="off"
+              value={this.state.form.fullName}
```

Comment:
Its good practice to destructure all your variables at the top of your function. Eg at the top of render do 

`const { fullName, _replyto, subject, message } = this.state.form`